simtools/util/general.py

```diff
@@ -647,3 +650,69 @@ def change_dict_keys_case(data_dict, lower_case=True):
         raise
 
     return _return_dict
+
+
+@u.quantity_input(rotation_angle=u.deg)
+def rotate(rotation_angle, x, y):
+    """
+    Rotate x and y by the rotation angle given in rotation_angle, in degrees.
+    The function returns the rotated x and y values in the same unit given.
+
+    Parameters
+    ----------
+    rotation_angle: astropy.units.deg
+        Angle to rotate the array in degrees.
+    x: numpy.array or list
+        x positions of the telescopes, usually in meters.
```

Comment:
The function seems to be generic (it is in `general.py`), but here it is about telescope positions? Why e.g., can't we replace  `model/camera.py::_rotate_pixels()` and have only one rotation function?

Reply:
Yes, that is the idea. Orel is going to check if that is possible. I will open an issue for that so we dont forget it.

Reply:
This is #438

Reply:
This should be possible, the only difference between the rotations is the direction. For the pixels the rotation is clockwise and here anticlockwise. The only question is whether we should add an option in this rotation function for the direction or should I change some code in the pixel rotations to accept also anticlockwise rotation (I think I prefer the former).
(will add the same comment to the issue.)